specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2016-05-01/Backup.json

```diff
@@ -0,0 +1,194 @@
+                    "description": "The name of the operation being performed on this particular object. This name should match the name that appears in RBAC or the event service.",
+                    "type": "string"
+                },
+                "display": {
+                    "description": "Contains the localized display information for this particular operation / action.",
+                    "$ref": "#/definitions/Display"
+                }
+            }
+        },
+        "Display": {
+            "description": "Contains the localized display information for this particular operation / action.",
+            "properties": {
+                "provider": {
+                    "description": "The localized friendly form of the resource provider name – it is expected to also include the publisher/company responsible. It should use Title Casing and begin with \"Microsoft\" for 1st party services. e.g. \"Microsoft Monitoring Insights\" or \"Microsoft Compute.\"",
+                    "type": "string"
+                },
+                "resource": {
+                    "description": "The localized, friendly version of the resource type related to this action or operation; the resource type should match the public documentation for the resource provider.",
+                    "type": "string"
+                },
+                "operation": {
+                    "description": "The localized, friendly name for the operation. Use the name as it will displayed to the user.",
+                    "type": "string"
+                },
+                "description": {
+                    "description": "The localized, friendly description for the operation. The description will be displayed to the user. It should be thorough and concise for used in both tooltips and detailed views.",
+                    "type": "string"
+                }
+            }
+        },
+        "OperationList": {
+            "description": "List of Operations",
+            "properties": {
+                "value": {
+                    "description": "Array of operations",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Operation"
+                    }
+                },
+                "nextLink": {
+                    "description": "URI to the next page of operations.",
+                    "type": "string"
+                }
+            }
+        },
+        "LongRunningOperationStatus": {
+            "description": "Status of a long running operation.",
+            "type": "object",
+            "properties": {
+                "code": {
+                    "description": "Operation result code.",
+                    "type": "string"
+                },
+                "message": {
+                    "description": "Description of the operation status.",
+                    "type": "string"
+                }
+            }
+        }
```

Comment:
Can we mark properties of Operation, Display and LongRunningOperationStatus as `readonly` also the properties OperationList.value, OperationList.nextLink?

Reply:
Done.